### PR TITLE
[SPARK-8925] [MLlib] Add @since tags to mllib.util

### PR DESCRIPTION
**Pull request by sthota2014**
Went thru the history of changes the file MLUtils.scala and picked up the version that the change went in.

### CODE REVIEW

**Comment by mengxr** [comment]
@sthota2014 You don't need to tag private or package private method. We only need `@since` on public methods.

**Comment by sthota2014** [comment]
@mengxr Removed the @since from the code for non-public methods.

**Comment by mengxr** [comment]
ok to test

**Comment by sthota2014** [comment]
@mengxr Any thoughts on this?

File: mllib/src/main/scala/org/apache/spark/mllib/util/MLUtils.scala [hunk 1]
```diff
@@ -114,7 +115,12 @@ object MLUtils {
 
   // Convenient methods for `loadLibSVMFile`.
 
-  @deprecated("use method without multiclass argument, which no longer has effect", "1.1.0")
```
Comment (mengxr) [inline mllib/src/main/scala/org/apache/spark/mllib/util/MLUtils.scala]: I think we need to leave the `@deprecated` outside the JavaDoc as a Scala annotation to make it work in Scala.

**Comment by sthota2014** [comment]
@mengxr Any thoughts on this ?

**Comment by sthota2014** [comment]
@mengxr can you please help me complete the cycle.

File: mllib/src/main/scala/org/apache/spark/mllib/util/MLUtils.scala [hunk 2]
```diff
@@ -133,7 +141,10 @@ object MLUtils {
       numFeatures: Int): RDD[LabeledPoint] =
     loadLibSVMFile(sc, path, numFeatures, sc.defaultMinPartitions)
 
-  @deprecated("use method without multiclass argument, which no longer has effect", "1.1.0")
```
Comment (mengxr) [inline mllib/src/main/scala/org/apache/spark/mllib/util/MLUtils.scala]: We still need to keep this as an annotation. `@deprecated` in comments is a JavaDoc tag, while `@deprecated` outside comments is a Scala annotation, which helps generate compiler warnings.

We should keep each PR minimal, so please keep all `@deprecated` annotation unchanged.

File: mllib/src/main/scala/org/apache/spark/mllib/util/MLUtils.scala [hunk 3]
```diff
@@ -114,6 +115,12 @@ object MLUtils {
 
   // Convenient methods for `loadLibSVMFile`.
 
+
+
+  /**
+   * @since 1.0.0
+   *
```
Comment (mengxr) [inline mllib/src/main/scala/org/apache/spark/mllib/util/MLUtils.scala]: Remove empty line.

**Comment by sthota2014** [comment]
@mengxr Any thoughts on this ?

**Comment by mengxr** [comment]
LGTM. Merged into master and branch-1.5. Thanks!

**Comment by rxin** [comment]
How come the Jenkins run here didn't trigger test failures?

**Pull request closed without merging**